curriculum/challenges/english/25-front-end-development/quiz-javascript-variables-and-data-types/66edc25ae5ea80bf6f785552.md

```diff
@@ -414,9 +414,8 @@ Which of the following is NOT a valid string concatenation method in JavaScript?
 
 ---
 
-```js
-console.log(`${string1} ${string2}`);
-```
+
+`console.log(`${string1} ${string2}`);`
```

Comment:
```suggestion
``console.log(`${string1} ${string2}`);``
```

note, as the code snippet contains backticks, you would need to use more backticks